src/Form/inputTypeConfig.js

```diff
@@ -9,15 +9,18 @@ const getDaxImg = isDDGApp || isFirefox ? daxBase64 : chrome.runtime.getURL('img
 
 /**
  * A map of config objects. These help by centralising here some of the complexity
- * @type {Object.<SupportedTypes, InputTypeConfig>}
+ * @type {Object<SupportedMainTypes, InputTypeConfig>}
  */
 const inputTypeConfig = {
     emailNew: {
         type: 'emailNew',
         getIconBase: () => getDaxImg,
         getIconFilled: () => getDaxImg,
         shouldDecorate: (isLogin, device) => device.hasLocalAddresses,
-        dataType: 'Addresses'
```

Comment:
As mentioned in the previous review, `emailNew` and `unknown` don't really use these, but I've added them for type consistency.